dashboard/templatetags/cards.py

```diff
@@ -222,41 +222,70 @@ def card_sleep_last(context, child):
 
 
 @register.inclusion_tag("cards/sleep_day.html", takes_context=True)
-def card_sleep_day(context, child, date=None):
+def card_sleep_day(context, child, end_date=None):
```

Comment:
Lets rename this card and template to just `sleep` or maybe `sleep_recent` -- you can decide (: